tests/test_auth.py

```diff
@@ -25,30 +36,30 @@ def setUp(self) -> None:
         self.url = reverse("admin:auth_user_export", kwargs={"object_id": self.user.id})
 
     def test_get_request_export_mixin(self):
-        resp = self.client.get(self.url)
-        self.assertEqual(302, resp.status_code)
+        resp = self.client.get(self.url, follow=True)
+        self.assertTemplateUsed(resp, "admin/login.html")
```

Comment:
Could you please check redirect url instead of template?

Reply:
Done